core/capabilities/launcher.go

```diff
@@ -46,6 +50,42 @@ type launcher struct {
 	subServices []services.Service
 }
 
+func unmarshalCapabilityConfig(data []byte) (capabilities.CapabilityConfiguration, error) {
```

Comment:
Which capability config is this unmarshalling? Not CCIP right?

Reply:
Not CCIP AFAIK. @cedric-cordenier right?

Reply:
Yeah this is only unmarshaling the Keystone capability configs